<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMusicTheoryMultiScaleDialog</class>
 <widget class="QDialog" name="QtMusicTheoryMultiScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicTheory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Games/GameBoenken/GameBoenken.qrc">
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #606060, stop: 1 #ffffff);
}

QLabel { font: 28px,  &quot;Courier&quot; }
QComboBox { font: 20px,  &quot;Courier&quot; }
QCheckBox { font: 20px,  &quot;Courier&quot; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ToolKalmanFilterer/ToolKalmanFilterer.qrc">:/images/PicRed.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="top_1">
       <item row="1" column="0">
        <widget class="QLabel" name="label_root_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Root</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="notes_1">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>C-D-E-F-G-A-B</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="scale_1">
         <item>
          <property name="text">
           <string>Major</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moll-Dur subdominant</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_notes_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_scale_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="root_1">
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ToolKalmanFilterer/ToolKalmanFilterer.qrc">:/images/PicGreen.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="top_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_notes_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_root_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Root</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="scale_2">
         <item>
          <property name="text">
           <string>Major</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moll-Dur subdominant</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="root_2">
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A#</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_scale_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="notes_2">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>C-D-E-F-G-A-B</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="display_major">
       <property name="text">
        <string>maj</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_minor">
       <property name="text">
        <string>min</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_aug">
       <property name="text">
        <string>aug</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_dim">
       <property name="text">
        <string>dim</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_m6">
       <property name="text">
        <string>m6</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="display_m7">
       <property name="text">
        <string>m7</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ribi::QtMultiScaleChordRelationsWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ribi::QtMultiScaleChordRelationsWidget</class>
   <extends>QWidget</extends>
   <header>qtmultiscalechordrelationswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Games/GameBoenken/GameBoenken.qrc"/>
  <include location="../ToolKalmanFilterer/ToolKalmanFilterer.qrc"/>
 </resources>
 <connections/>
</ui>
